components/create_comment/create_comment.jsx

```diff
@@ -883,7 +884,7 @@ export default class CreateComment extends React.PureComponent {
         let emojiPicker = null;
         const emojiButtonAriaLabel = formatMessage({id: 'emoji_picker.emojiPicker', defaultMessage: 'Emoji Picker'}).toLowerCase();
 
-        if (this.props.enableEmojiPicker && !readOnlyChannel) {
+        if (this.props.enableEmojiPicker && !readOnlyChannel && !this.state.preview) {
             emojiPicker = (
                 <div>
                     <EmojiPickerOverlay
```

Comment:
nit: Not related to this PR but was from recent commit (https://github.com/mattermost/mattermost-webapp/pull/3136/files) but can you change the name of state, from `preview` to `showPreview`? It's a bit confusing since on render function we have another `preview` variable.

Same case for `CreatePost` component